src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -120,6 +127,11 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
         }
         $this->classDiscriminatorResolver = $classDiscriminatorResolver;
         $this->objectClassResolver = $objectClassResolver;
+
+        if (null === $instantiator) {
+            throw new InvalidArgumentException(sprintf('Instantiator parameter is required, please use "%s" to instantiate your Normalizer.', get_parent_class($this).'Factory'));
```

Comment:
This is a BC break. Cannot you initialize a new `Instantiator` is none is provided?

Reply:
This is why I can't finish this PR actually, `Instantiator` needs to come from the DIC and can't be initialized inside the construct method because we need the DIC to inject a denormalizer inside it.
In the last PR, I rewrote the `setDenormalizer` method to make this behavior, but this is totaly not good ~ (see your comment there https://github.com/symfony/symfony/pull/30956/files#r380091247)
If you have any idea how I can solve this issue feel free to help here :+1: